pkg/config/config.go

```diff
@@ -19,25 +19,26 @@ type Config struct {
 
 	OutputPath string `yaml:"OutputPath"`
 
-	EnableLog bool `yaml:"EnableLog"`
-	LogLevel  int  `yaml:"LogLevel"`
+	ListRefereces    bool `yaml:"ListRefereces"`
```

Comment:
Misspelling: Refereces -> References

Reply:
Resolved in [76b3090](https://github.com/AntonSkrub/GoGit-Integration/pull/6/commits/76b3090a4275bf0725703d0684da0ca154dd884f)